### ISSUE TITLE
Setup help for NEWBIES

### ISSUE THREAD

**Issue by felixvos** [MMM-CalendarExt2/MMM-CalendarExt2#131]
Hi! This looks like a great plug in for MagicMirror! I am VERY new to coding/using the type of language needed in the config.js section so I was wondering if someone can help? 

I have installed the MMM-MonthlyCalendarExt2 to my MM modules and that seemed to work.

My ultimate goal is to connect my google calendar(s) to this monthly view for the MM. I am trying to see the ENTIRE month with days of the week at the top (Sunday-Saturday) along with the Month name at the top, dates in each box, and for it to display the events and times that are on each day. Would love to be able to have multiple calendars included in this view differentiated by colors as google calendar does.

I read the configuration docs but since I am so new to this stuff, it is hard to follow and execute using my google calendars.

Do you by chance have a video or step by step guide using pictures of what to do to achieve my goal? Any help would be great.

**Comment by klaernie** [comment]
Hi @felixvos,

well, since the requirements are very different for different people, making a step-by-step guide is really hard to do. But there is an example in the docs/examples folder already: https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/master/docs/examples/klaernie.md (and yes, it's my personal setup as of quite a while ago).

I'd suggest you to just try, and copy something and play around with it. Then if there is something specific that you can't get to work ask a more detailed question. I'd say take my example, and remove everything you don't need ;)

**Comment by felixvos** [comment]
Thanks so much! Totally understand what you mean about step by step. I missed that example. I will try to tinker using that and come back if I have questions.

**Comment by klaernie** [comment]
No problem. And once you've got it running feel free to submit your example as well - I'm sure someone will be happy later on to discover a different approach!

**Comment by felixvos** [comment]
Hi there. I was able to leverage your example and tried to build out my own. I added my 4 google calendars and am running into some issues and have some questions.

I tested one calendar at first and it was appearing in the MM but when I added the other 3, the original calendar that I added was no longer showing up and only 1 of the other 3 that I added are appearing. **So the config includes all 4 calendars and when I launch MM only 1 show up**. Possible issue I think could play into this is below.

I am using 4 google calendars, 2 have ical secret links and 2 did not (assuming because they were shared with me). So what I did was took the public link and copied everything after the "src=" through the google.com and pasted that into this format where the *********** are to make an ics link **https://calendar.google.com/calendar/ical/**********/public/basic.ics**. 
If this is an issue, if I get the secret iCal url from the person who owns the shared calendar will that fix this issue?

That being said, when I commented out the 2 calendars that I did the above to (did not have the original ics link) and only left the 2 calendars that actually had an iCal secret url in google, it still only shows me one of them and I can't figure out how to have both of them appear. I went into the css and gave each of those a different colored background to see if that would solve anything and the css formatting works for one of the calendars but not the other.

Another issue I have is when MM is launched the calendar fades as if its updating or something but I don't have anything set to update. Any way to stop the fading/fake updating look?

Please see my config for the MMM-CalendarExt2 module below. Maybe you know how to resolve this issue so all the calendars will appear?

{
			module: "MMM-CalendarExt2",
			config: {
			calendars : [
			{
				name: "Our Calendar",
				url: "work-around iCal url",
			},
			{
				name: "Name Calendar",
				url: "iCal url",
			},
			{
				name: "Holidays",
				url: "iCal url",
			},
			{
				name: "Birthdays",
				url: "work-around iCal url",
			},
			
						],
			views: [
			{
				name: "Overview Calendar",
				title: "Overview Calendar",
				mode: "week",
				slotCount: 4,
				slotMaxHeight: "120px",
				hideoverflow: false,
				slotAltTitleFormat: "MMM/D",
				position: "fullscreen_below",
				calendars: ["Our Calendar", "Name Calendar", "Birthdays", "Holidays"],
			},
					],
			scenes: [
				{
				name: "DEFAULT",
				},
					],
					},
		},

![image](https://user-images.githubusercontent.com/8019378/147687666-cba188a4-59dd-4752-a01d-e9ee5608ea6c.png)

**Comment by klaernie** [comment]
Starting one problem at a time:
- Can you please verify that the URLs you use in the config return a valid ICS? Open the URLs in an incognito windows or use curl/wget on the command line and then take a look if the resulting file looks valid.
- What does the log written by Magic Mirror say? Does it list that it read events from each of the calendars?
- What does the console in the frontend say?

**Comment by KristjanESPERANTO** [comment]
Since there hasn't been any further feedback in quite some time, I'm closing this issue now to keep the tracker tidy. Feel free to reopen or create a new issue if you're still experiencing problems 🙂
